### DISCUSSION TITLE
Add ability to search in org's private repo or repo that a user have access to

### DISCUSSION THREAD

**Discussion by martgil** [community/community#10526] [Other Feature Feedback, Questions, & Ideas]
Code Search is great so far!

I only miss the ability to search from repositories that an org has given me access to.

I hope it would gradually be implemented  :)

**Comment by colinwm** [comment]
Hi @martgil, glad you're enjoying code search! 

Can you explain what you mean by "repositories that an org has given you access to"? All repositories you have access to (public and private) _should_ be indexed and accessible via cs.github.com.

Did you find some repositories that didn't get indexed?

**Reply by austenstone** [reply]
I think he's looking for the ability to filter repos that he has write access to.

The `team > repos` page might help

**Reply by domdfcoding** [reply]
Is there a setting I'm supposed to change in the private repository because none of them are indexed?
